examples/audio/audio_control.rs

```diff
@@ -0,0 +1,65 @@
+use bevy::{audio::AudioSink, prelude::*};
+
+/// This example illustrates how to load and play an audio file, and control how it's played
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup)
+        .add_system(update_speed)
+        .add_system(pause)
+        .add_system(volume)
+        .run();
+}
+
+fn setup(
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    audio: Res<Audio>,
+    audio_sinks: Res<Assets<AudioSink>>,
+) {
+    let music = asset_server.load("sounds/Windless Slopes.ogg");
+    let handle = audio_sinks.get_handle(audio.play(music));
+    commands.insert_resource(MusicControler(handle));
+}
+
+struct MusicControler(Handle<AudioSink>);
```

Comment:
```suggestion
struct MusicController(Handle<AudioSink>);
```

I won't make this suggestion in the entire file, but that typo is repeated in each system